### PR TITLE
Updated dependabot config

### PR DESCRIPTION
**Pull request by Fuzznimp**
## Description

This PR aims at cleaning up the ignored lib version added by the automated migration tool from GitHub (from `v1` to `v2`).
I took the time to fine-tuned it, and add the GitHub Actions version checks.

## Type of change

- [x] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## Checklist

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -4,81 +4,20 @@ updates:
   directory: "/"
   schedule:
     interval: daily
-    time: "04:00"
+    time: "10:00"
+    timezone: "Europe/Paris"
+  open-pull-requests-limit: 10
+  reviewers:
+  - Fuzznimp
+  - Oluane
```
Comment (Meyclem) [inline .github/dependabot.yml:12]:
```suggestion
  reviewers:
  - Fuzznimp
  - Oluane
  - Meyclem
```
😁

Comment (Oluane) [inline .github/dependabot.yml:12]: Maybe we could add Meyclem here too ?

Comment (Fuzznimp) [reply .github/dependabot.yml]: Done

Comment (Fuzznimp) [reply .github/dependabot.yml]: Done

**Review by Meyclem — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (Oluane) [reply .github/dependabot.yml]: You beat my comment by 17 seconds... 😆

**Review by Oluane — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (Meyclem) [reply .github/dependabot.yml]: Not quick enough little beetle 😁

Comment (Fuzznimp) [reply .github/dependabot.yml]: 😆

**Pull request merged**